README.md

```diff
@@ -1,51 +1,20 @@
-## GPII Preferences Server Dockerfile
+## GPII Preferences Server Dockerfile - Ansible version
 
+Builds a GPII Preference Server Docker container image. The image is built using the [Ansible role](https://github.com/gpii-ops/ansible-gpii-preferences-server).
```

Comment:
This might be nitpicky but would it be possible to change ``ansible-gpii-preferences-server`` to ``ansible-preferences-server``? For brevity's sake we've been leaving out ``gpii`` out of naming schemes for ansible and docker projects with the exception of ``gpii-framework``.